src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -265,6 +265,21 @@ public function addCasters(array $casters): void
         }
     }
 
+    /**
+     * Adds default casters for resources and objects.
+     *
+     * Maps resources or objects types to a callback.
+     * Types are in the key, with a callable caster for value.
+     * Resource types are to be prefixed with a `:`,
+     * see e.g. static::$defaultCasters.
+     *
+     * @param callable[] $casters A map of casters
+     */
+    public static function addDefaultCasters(array $casters): void
+    {
+        self::$defaultCasters += $casters;
```

Comment:
`+=` won't override the default casters that already have been set. Is it intended?